### ISSUE TITLE
Works until the math gets too complicated

### ISSUE THREAD

**Issue by Cahleeeb** [survivalizeed/4-equals-10-auto-solver#1]
Doesn't deal with negatives well, seems to not be able to do parenthesis, no way to exclude expressions.

**Comment by survivalizeed** [comment]
Yes I know. I'm going to implement this when I return from my vacation. Thanks for testing!

**Comment by survivalizeed** [comment]
> Doesn't deal with negatives well, seems to not be able to do parenthesis, no way to exclude expressions.

Fixed in the Julia version
